native/src/components/EventListItem.tsx

```diff
@@ -14,6 +24,22 @@ const Description = styled.Text`
 
 const placeholderThumbnails = [EventThumbnailPlaceholder1, EventThumbnailPlaceholder2, EventThumbnailPlaceholder3]
 
+const getDateIcon = (date: DateModel): { icon: JSXElementConstructor<SvgProps>; label: string } | null => {
```

Comment:
I think this is not possible (or rather: not easily possible). First, assets are in a separate folder and not easily usable as in web/native. Second, the types in web and native differ. In web, the icon type is just a string while in native it is JSXElementConstructor<SvgProps> because of the different ways we render svgs. Do you have any idea how to do this @ztefanie anyway?

Reply:
See my suggestion here: https://github.com/digitalfabrik/integreat-app/pull/2512